apps/src/studio/api.js

```diff
@@ -121,6 +121,10 @@ exports.changeScore = function(id, value) {
   Studio.queueCmd(id, 'changeScore', {'value': value});
 };
 
+exports.getScore = function () {
```

Comment:
blockly functions require an `id` parameter so that block highlighting works. The APIs should route through `Studio.queueCmd()` for consistency as well.